src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagEmailService.kt

```diff
@@ -76,14 +79,17 @@ class ProductaanvraagEmailService @Inject constructor(
     ) {
         mailTemplateService.findMailtemplateByName(automaticEmailConfirmation.templateName)?.let {
                 mailTemplate ->
+            val replyToAddress = automaticEmailConfirmation.emailReply?.let {
+                MailAdres(it.extractEmailAddress(configuratieService), null)
+            }
             val mailGegevens = MailGegevens(
-                MailAdres(automaticEmailConfirmation.emailSender, null),
-                MailAdres(to, null),
-                MailAdres(automaticEmailConfirmation.emailReply, null),
-                mailTemplate.onderwerp,
-                mailTemplate.body,
-                null,
-                true
+                from = MailAdres(automaticEmailConfirmation.emailSender.extractEmailAddress(configuratieService), null),
```

Comment:
Checking: should the second argument (`name`) really be null? Don't we have the name?

And certainly in case of `null` arguments I think it is nicer to use named parameters to increase readbiality. 
```suggestion
                from = MailAdres(
                  email  = automaticEmailConfirmation.emailSender.extractEmailAddress(configuratieService), 
                  name = null
                  ),
```

Reply:
To retrieve the name, we would likely need to make additional calls to open klant, such as `klantClientService.listBetrokkenenByNumber()`, along with some extra logic to extract the contact name for both NatuurlijkPersoon and NietNatuurlijkPersoon. Since this is not part of the acceptance criteria for the current story, we could extract it into a separate improvement story.

Reply:
Ok. I do think you need to check with the PO about the email name though. And check if and how we use the name for the existing manual email confirmations. Can you do that?

PS: for GEMEENTE you can just get the name as follows: `configuratieService.readGemeenteNaam()`. There is even already a function for this: `MailService.getGemeenteMailAdres`. I'm surprised you are not using that function already here. Is there a reason for that?

Reply:
But indeed it is fine to do the names in a follow up story. Can you check this with the PO and create a story for it?